build.sbt

```diff
@@ -40,19 +42,77 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.1")
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.9.2")
 addSbtPlugin("com.typesafe.sbt" % "sbt-gzip" % "1.0.0")
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.1.6")
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC2")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC3")
```

Comment:
v1.0.0-RC6  is out, final release is coming soon.  Please add a todo in code to replace and new ticket to update in jira project.